libs/SmartCard/inc/StorageClassISO15693.h

```diff
@@ -68,6 +73,15 @@ Module Name:
 #define ISO15693_UIDBYTE_5_STM_M24LR64ER    0x5C
 #define ISO15693_UIDBYTE_5_STM_M24LR16ER    0x4C
 
+#define ISO15693_FLAG_PROTOEXT              0x08U
+/* Check if protocol extension bit is needed in the request flag */
+#define ISO15693_PROTOEXT(pUid) \
+    ((ISO15693_MANUFACTURER_STM == pUid[ISO15693_UID_BYTE_6]) && \
+        ((pUid[ISO15693_UID_BYTE_5] & ISO15693_UIDBYTE_5_STM_MASK) == ISO15693_UIDBYTE_5_STM_LRIS64K || \
+         (pUid[ISO15693_UID_BYTE_5] & ISO15693_UIDBYTE_5_STM_MASK) == ISO15693_UIDBYTE_5_STM_M24LR64R || \
+         (pUid[ISO15693_UID_BYTE_5] & ISO15693_UIDBYTE_5_STM_MASK) == ISO15693_UIDBYTE_5_STM_M24LR64ER || \
+         (pUid[ISO15693_UID_BYTE_5] & ISO15693_UIDBYTE_5_STM_MASK) == ISO15693_UIDBYTE_5_STM_M24LR16ER))
```

Comment:
Shouldn't ideally this criteria be generic, manufacturer-independent and based on solely ISO15693 spec?

Reply:
ISO15693_PROTOEXT in libs/SmartCard/inc/StorageClassISO15693.h = ISO15693_PROTOEXT_FLAG_REQUIRED in phFriNfc_ISO15693Map.h